OPHD/States/PlanetSelectState.cpp

```diff
@@ -23,6 +23,17 @@ std::size_t planetSelection;
 constexpr std::size_t planetSelectionInvalid = std::numeric_limits<std::size_t>::max();
 
 
+namespace
+{
+	std::vector<Planet::Attributes> PlanetAttributes =
+	{
```

Comment:
This is a little bit off-topic for this PR, but:
Technically you can get rid of the `=`. It would then be using brace initialization syntax `{}`.

----

We might be using capitalization inconsistently when it comes to constants. I think this was a discussion that came up about a year ago.

You can initialize local variables with runtime results from functions (non-`constexpr`), and declare them as `const`, and they typically won't be capitalized. You can also initialize them with `constexpr` compile-time known values and declare them as `const`. These are pretty similar to the previous case, so we might also use lowerCamelCase. We can also move such variables to a top level global position. But should we be using a different naming convention based on whether a variable is declared locally or globally?
```cpp
void someFunction()
{
    auto mutableValue = someOtherFunction1();
    const auto runtimeValue = someOtherFunction2(); // Not constexpr compatible
    const auto anotherConst = 5; // Or should this be "AnotherConst" (constexpr compatible)
}

const auto yetAnotherConst = 6; // Or should this be "YetAnotherConst" (same, constexpr compatible, but global)
```

I'm inclined to use a consistent naming convention regardless of scope.

I'm also maybe a little concerned that constant values could be mistaken for `class` names. Occasionally they may sound pretty similar.

On the other hand, the `m` prefix is scope based. Or should we consider a `g` prefix?

Reply:
The naming convention I've used is this:

```cpp
mVarName; // Class member variable
functionName(); // member or stand-alone function
ClassName;
ConstantName;
EnumName;
localVariable;
```

I used to declare constants `IN_ALL_CAPS` but you've discouraged that in favor of `UpperCamelCase` and I'm finding that I prefer the latter vs the former so new code is using that approach.

I've noticed inconsistent capitalization in code that I haven't written myself and it's confused me a number of times because something like `notActuallyLocal` is declared as module or global constant when it should have been `NotActuallyLocal`. And so on.

Something I'll be addressing moving forward and being more strict about when accepting PR's

Reply:
I know this is nitpicking a bit, but you mention `ConstantName` and `localVariable`, but a variable can be both constant and local. :wink: